docs/advanced-features/new-ai-plugin-usage.md

```
@@ -0,0 +1,225 @@
+工具调用中(以查询天气Mock工具为例)：
+![MCP工具调用示例中](./imgs/ai-assistant-tool-execution1.png)
+工具调用完成后会返回最终结果，也可以点击展开工具调用参数与结果：
+![MCP工具调用示例结果示例](./imgs/ai-assistant-tool-execution2.png)
+
+## 五、典型使用场景
+
+### 5.1 页面搭建场景 (即将开放)
+
+用户可以通过自然语言描述来生成页面：
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)